src/SessionManager.php

```diff
@@ -133,6 +133,10 @@ public function start($preserveStorage = false)
         if ($oldSessionData instanceof \Traversable
             || (! empty($oldSessionData) && is_array($oldSessionData))
         ) {
+            if(! is_array($oldSessionData)) {
+                $oldSessionData = $oldSessionData->toArray();
```

Comment:
This assumes $oldSessionsData from $_SESSION superglobal is an object that implements `toArray()` method.
Can it be something else?